<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Retriever</class>
 <widget class="QMainWindow" name="Retriever">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Retriever</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="taskList"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonNewTask">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonEditTask">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDeleteTask">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSyncTask">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Sync</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSyncSelectedTask">
          <property name="text">
           <string>Sync Selec.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="console">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>359</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuFileSave"/>
    <addaction name="separator"/>
    <addaction name="menuFileExportSettings"/>
    <addaction name="menuFileImportSettings"/>
    <addaction name="separator"/>
    <addaction name="menuOptionsShow"/>
    <addaction name="menuFileQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menuHelpAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="menuFileQuit">
   <property name="icon">
    <iconset>
     <disabledon>:/icons/resources/TangoIcons/process-stop.png</disabledon>
    </iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="menuHelpAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="menuFileSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="menuOptionsShowFullLogs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Full Logs</string>
   </property>
  </action>
  <action name="menuHelpPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="menuOptionsShowLogWindow">
   <property name="text">
    <string>Show Log Window</string>
   </property>
  </action>
  <action name="menuOptionsShow">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="menuFileExportSettings">
   <property name="text">
    <string>Export Settings...</string>
   </property>
  </action>
  <action name="menuFileImportSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import Settings...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
